packages/atomic/src/components/facets-v1/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -120,6 +129,15 @@ export class AtomicNumericFacet
     };
   }
 
+  private buildManualRanges(): NumericRangeWithLabel[] {
+    return Array.from(this.host.querySelectorAll('atomic-numeric-range')).map(
+      ({start, end, endInclusive, label}) => ({
```

Comment:
How protected are we if the user "badly" configures their HTML for `atomic-numeric-range` ?

Reply:
If it doesn't pass Headless validation, an error will be thrown from Headless, and the error component displayed in the UI